assets/js/plugin-check-admin.js

```diff
@@ -13,6 +13,21 @@
 		return;
 	}
 
+	// Handle disabling the check it button when a plugin is not selected.
+	pluginsList.addEventListener( 'change', canRunChecks );
+
+	function canRunChecks() {
+		if ( '' === pluginsList.value ) {
+			checkItButton.disabled = true;
+		} else {
+			checkItButton.disabled = false;
+		}
+	}
```

Comment:
Not directy affiliated with this PR, but now that we're adding this logic to deactivate the button, I see there is a place where our logic is somewhat problematic: If you start running checks, the button gets disabled, but you can still change the plugin dropdown to select another plugin, which would unset the `disabled` on the button.

Could you modify the relevant logic below to also set the dropdown to `disabled` while checks are running (and then unset `disabled` also on the dropdown once checks are done)?

Reply:
Thanks @felixarntz I've updated to disable the plugin list while checks are running, then re-enable once the checks are complete.